langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -333,6 +473,13 @@ public isolated function elementChildren(xml x, string? nm = ()) returns xml<Ele
 #
 # Each item is represented as a singleton value.
 #
+# ```ballerina
+# xml books = xml `<book>Java</book><book>Ballerina</book>`;
+# books.map(function (xml xmlContent) returns xml {
+#     return xml `<book author="Anjana">${xmlContent.children()}</book>`;
+# }) ⇒ <book author="Anjana">Java</book><book author="Anjana">Ballerina</book>
```

Comment:
Better to use kind or something as the attribute instead of author.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/9dc46b7c36c28b92b9ce360ed71621b3a6eff8c0